osu.Game/Online/Rooms/RoomCategory.cs

```diff
@@ -1,12 +1,12 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-namespace osu.Game.Online.Multiplayer
+namespace osu.Game.Online.Rooms
 {
     public enum RoomCategory
     {
         Normal,
```

Comment:
Not sure what the `Normal` category is supposed to be now - maybe it should be `Playlist`, `SpotlightPlaylist`, `Multiplayer`?

Reply:
actually you make a good point - we can't change this without extra handling for osu-web api type mapping. will probably need to revert this one back.